csrc/runtime/allocations.cpp

```diff
@@ -524,7 +524,7 @@ class BackwardTraverseFromAllocToLogical {
         new_shape.emplace_back(tensor_.size(i));
       }
     }
-    tensor_ = tensor_.view(new_shape);
+    tensor_ = tensor_.reshape(new_shape);
```

Comment:
Yeah, I know I was the one who did this, but I haven't really spent time on whether this is a legit thing to support here.

I'm scared that the `reshape` here might be suggesting. Since the case we are trying to support with block layout isn't representable with lower dimensional pytorch tensor.

Reply:
Based on offline discussion - moved to using `as_strided`.